test/data/mobility/br/sample_mobility_matrix.csv

```diff
@@ -0,0 +1,40 @@
+DATE,ORIGIN,DESTINATION,COUNT
```

Comment:
There actually already is a line with a missing value. I just don't have a test to check the logged warning. Would like to figure out how to log all errors at the end of the program and test there.